modal/image.py

```diff
@@ -666,13 +666,13 @@ def build_dockerfile(version: ImageBuilderVersion) -> DockerfileSpec:
             context_mount=mount,
         )
 
-    def _add_local_python_packages(self, *packages: Union[str, Path], copy: bool = False) -> "_Image":
+    def add_local_python_dir(self, module_name: str, *, copy: bool = False) -> "_Image":
```

Comment:
Just being annoying but is it a problem that "module" technically corresponds to a [single file](https://docs.python.org/3/tutorial/modules.html)?

Reply:
It's definitely a problem... The naming of this function and argument is super hard :(

A single file module could also be considered a package right, if it's the only top level entry of a project. So maybe `add_local_python_package`(+-s) is good after all? But it's still potentially confusing since it doesn't "install" anything...